src/main/webapp/WEB-INF/tags/answerWith.tag

```diff
@@ -7,7 +7,7 @@
 <%@attribute name="commentVotes" type="org.mamute.model.CommentsAndVotes" required="true" %>
 <section class="post-area ${answer.isVisibleForModeratorAndNotAuthor(currentUser.current) ? 'highlight-post' : '' }">
 	<div class="post-meta">
-		<tags:voteFor item="${answer}" type="${t['answer.type_name']}" vote="${vote}"/>
+		<tags:voteFor item="${answer}" type="${answer.typeName}" vote="${vote}"/>
```

Comment:
@leocwolter: The other places are indeed localized URL parameters which are then resolved on server-side using caelum/mamute/blob/master/src/main/java/org/mamute/infra/ModelUrlMapping.java

Is there anything else holding up the merge of this PR?